print-arista.c

```diff
@@ -93,17 +93,13 @@ arista_print_date_hms_time(netdissect_options *ndo, uint32_t seconds,
 		uint32_t nanoseconds)
 {
 	time_t ts;
-	struct tm *tm;
-	char buf[BUFSIZE];
+	char buf[128];
```

Comment:
OK. Thus `#include "addrtoname.h"` can be removed.

Reply:
Removed.  I also changed `128` to `sizeof(<example string>)`.